CMakeLists.txt

```diff
@@ -29,7 +29,7 @@
 # https://www.khronos.org/registry/spir-v/
 #
 cmake_minimum_required(VERSION 3.0)
-project(SPIRV-Headers VERSION 1.4.1)
+project(SPIRV-Headers VERSION 1.5.1)
```

Comment:
What does this get used for?  I'd rather eliminate it than have yet another independent version number that must be kept in sync.